src/helpers/auth.js

```diff
@@ -2,13 +2,16 @@ import { firebaseAuth, provider } from '../config/firebase-config';
 
 export default {
 	logIn: () => {
-		return firebaseAuth.signInWithPopup(provider).then((result) => {
-			// const token = result.credential.accessToken;
-			// const user = result.user;
+		firebaseAuth.signInWithRedirect(provider);
+	},
+
+	auth: () => {
+		return firebaseAuth.getRedirectResult().then(function(result) {
 			return result;
-		}).catch((error) => {
-			console.log(error.code);
+		}).catch(function(error) {
+			console.log(error)
 		});
+
```

Comment:
i see that after creating the class, you instantiated it and then exported the instance [here](https://github.com/iakhator/news-feed-application/blob/develop/src/helpers/auth.js#L18-L19). But why?